.travis.yml

```diff
@@ -6,13 +6,13 @@ os:
 - linux
 dist:
 # take the latest supported LTS version
-- trusty
+- xenial
```

Comment:
why not Ubuntu 18.04 ? that is the latest LTS available

Reply:
on travis, only 12/14/16.04 are available (it's always a bit behind).
I would actually like to also test on trusty still (so I'll keep this in mind, just doesn't work with R), but since standard support for it has stopped last month anyway, it's maybe no problem to move on.